config/locales/diaspora/en.yml

```diff
@@ -812,6 +812,27 @@ en:
         Cheers,
 
         The diaspora* email robot!
+    export_photos_email:
+      subject: "Your photos are ready for download, %{name}"
+      body: |-
+        Hello %{name},
+
+        Your photos have been processed and are ready for download by following [this link](%{url}).
+
+        Cheers,
+
+        The diaspora* email robot!
+    export_photos_failure_email:
+      subject: "We’re sorry, there was an issue with your photos, %{name}"
```

Comment:
I'd suggest rewording this title to 'There was a problem exporting your photos', move the 'sorry' bit to the end of the email and remove the user name from the title.